src/main/kotlin/io/snyk/plugin/ui/toolwindow/SnykToolWindowPanel.kt

```diff
@@ -362,16 +363,18 @@ class SnykToolWindowPanel(val project: Project) : JPanel(), Disposable {
                     )
                 }
                 is IacIssueTreeNode -> {
+                    val iacIssuesForFile = (node.parent as? IacFileTreeNode)?.userObject as? IacIssuesForFile
+                        ?: throw IllegalArgumentException(node.toString())
+                    val fileName = iacIssuesForFile.targetFilePath
+                    val virtualFile = VirtualFileManager.getInstance().findFileByNioPath(Paths.get(fileName))
+                    val psiFile = virtualFile?.let { PsiManager.getInstance(project).findFile(it) }
+
                     val iacIssue = node.userObject as IacIssue
                     val scrollPane = wrapWithScrollPane(
-                        IacSuggestionDescriptionPanel(iacIssue, project)
+                        IacSuggestionDescriptionPanel(iacIssue, psiFile, project)
```

Comment:
@ArtsiomCh, are we ok with passing `psiFile`? Could we get it from iacIssue in the detail panel?